models/games.model.js

```diff
@@ -4,6 +4,17 @@ exports.selectCategories = () => {
   return db.query("SELECT * FROM categories;").then(({ rows }) => rows);
 };
 
+exports.selectReviews = () => {
+  return db
+    .query(
+      `SELECT owner, title, reviews.review_id, review_img_url, reviews.votes, category, owner, reviews.created_at, designer,
```

Comment:
got two 'owner' in the query string. Also better to keep the format reviews.owner